articles/quickstart/spa/angular-beta/02-calling-an-api.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Calling an API
+description: This tutorial demonstrates how to call a backend API using an access token.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - angular
+  - login
+github:
+    path: 02-Calling-an-API
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD006 MD002 MD041 -->
+
+Most single-page apps use resources from data APIs. You may want to restrict access to those resources, so that only authenticated users with sufficient privileges can access them. Auth0 lets you manage access to these resources using [API Authorization](/api-auth).
+
+This tutorial shows you how to create a simple API using [Express](https://expressjs.com) that serves resources protected by a middleware that looks for and validates access tokens. You will then see how to call this API using an access token granted by the Auth0 authorization server.
+
+<%= include('../_includes/_calling_api_create_api') %>
+
+<%= include('../_includes/_calling_api_create_backend.md') %>
+
+Finally, modify `package.json` to add two new scripts `dev` and `server` that can be used to start the frontend and the backend API together:
+
+```json
+"scripts": {
+  "ng": "ng",
+  "start": "ng serve",
+  "build": "ng build",
+  "test": "ng test",
+  "lint": "ng lint",
+  "e2e": "ng e2e",
+  "server": "node server.js",
```

Comment:
@joshcanhelp This is tricky to do as JSON doesn't support comments. Comments are best used when the language itself supports them, so that when the code is copied out it can slot straight into their file without problems.

Of course, I _can_ put comments into the snippet; do you think it would be ok to have to alter the code after commenting so that it doesn't break, or would you rather just be able to copy as-is?

Reply:
That makes sense!